nginx/Dockerfile

```diff
@@ -6,7 +6,10 @@ ENV NODE_PORT=$NODE_PORT
 ARG REACT_PORT
 ENV REACT_PORT=$REACT_PORT
 
+ARG SERVER_NAME
+ENV SERVER_NAME=$SERVER_NAME
```

Comment:
the expansion works fine here? :thinking: 
`$SERVER_NAME` and not `${SERVER_NAME}`? :yum:

Reply:
Yesyes no problem no need here!

Reply:
coool :ok_hand: 
better than `bash` then :eyes: